### PR TITLE
Add GTK Style Class to a context (right-click) menu

### PR DESCRIPTION
**Pull request by nefigtut**
This fixes an issue when a monospace font is displayed instead of a proper one in mate-terminal context (right-click) menu. This PR tries to bring a fix from Ubuntu to the upstream.

This fixes: https://github.com/mate-desktop/mate-terminal/issues/407

Also see a number of related bug reports:
https://bugs.launchpad.net/ubuntu/+source/mate-terminal/+bug/1955505
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=998856
https://groups.google.com/g/linux.debian.bugs.dist/c/Xa9ZBXh_1nE
https://bugzilla.redhat.com/show_bug.cgi?id=2160523

### CODE REVIEW

**Review by lukefromdc — APPROVED** [review]
Had to test this in Menta as with my normal theme the difference was too subtle for me to see but it definately works. Years ago I and several others had to deal with a similar issue in Pluma and didn't find this simple solution

**Comment by lukefromdc** [comment]
Hoping we can get a second review on this

**Review by zhuyaliang — APPROVED** [review]
The problem was fixed

**Comment by nefigtut** [comment]
thank you, @zhuyaliang

**Comment by nefigtut** [comment]
thank you, @zhuyaliang, @lukefromdc, most appreciated.

**Comment by raveit65** [comment]
I guess this can be cherry-picked to 1.26 as a bugfix.

**Pull request merged**
